certified-connectors/KyndrylMainframe/readme.md

```diff
@@ -0,0 +1,35 @@
+# Kyndryl Mainframe Connector
```

Comment:
@RTkyndryl please follow our template for the readme file [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).